exporter.go

```diff
@@ -366,6 +398,35 @@ func (b *Exporter) handleEvent(event Event) {
 			conflictingEventStats.WithLabelValues("counter").Inc()
 		}
 
+	case *FlushEvent:
```

Comment:
I'm not sure about handling the flush in-line with incoming statsd events. It's fundamentally different (internal, not user-generated. Would it make sense to break it out into its own loop, or does that get us into locking/mutex hell?

Reply:
if we keep it inline, please add a comment explaining why it's here

Reply:
I'm not sure how much locking/mutex work there would be, I'm not super familiar with your code.   This mechanism let me completely avoid the question.  :)

Reply:
you would need to mutex-protect all the map accesses. it's fine this way, but I think we can't safely circle back into `handleEvent` because that would cause double-mapping.